api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt

```diff
@@ -0,0 +1,117 @@
+package bomber.arango.ArangoDao
+
+import com.arangodb.ArangoCollectionAsync
+import com.arangodb.ArangoDBAsync
+import com.arangodb.ArangoDatabaseAsync
+import com.arangodb.entity.BaseDocument
+import com.arangodb.model.AqlQueryOptions
+import com.arangodb.util.MapBuilder
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+
+/**
+ * This dao is responsible for crud operation scenarious
+ * @author kostya05983
+ */
+class JsonScenarioDao {
```

Comment:
maybe we will call json in uppercase as JSON. Identical similar objects: IP, DAO, ID,

Reply:
I can this rename,and use it as the name of class, but in java we rarely call json as JSON , but Dao I rename in DAO